### DISCUSSION TITLE
Live update prefix in a different form Section schema

### DISCUSSION THREAD

**Discussion by darpadev** [filamentphp/filament#12704] [Help]
### Package

Form builder

### Package Version

v3.2.72

### How can we help you?

![image](https://github.com/filamentphp/filament/assets/37233788/71019a0c-2b00-42e8-a632-eecf910e3e7f)

1. I have a form enabling users to choose their preferred `currency` (the top red box)
2. When the currency is being selected/changed, I want it to appear in the `price` field (the bottom red box) as a prefix

Current situation:
- When I create the form just as shown in the snapshot, the `currency` field does not update the `price`'s prefix.
- However, if I put the `currency` field in the same schema as the `price` field, the `price`'s prefix does update when the `currency` is selected.

Any idea on how to make the `currency` field to be able to update the `price`'s prefix while maintaining the current field arrangement as shown in the snapshot? Because the `price` field is in a repeating panel, and making the user have to select a currency each time a new panel is created doesn't seems a good experience

**Comment by leandrocfe** [comment]
Add `live()` in the currency field

Add in the price
```php
->prefix(fn (Get $get): string => $get('currency'))
```

**Reply by bilogic** [reply]
https://filamentphp.com/docs/3.x/forms/fields/repeater#using-get-to-access-parent-field-values

repeater requires `../../currency`
